PgpWordList/PgpWordAggregationBuilder.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Messerli.PgpWordList
+{
+    public class PgpWordAggregationBuilder
+    {
+        public const string DefaultSeparator = "-";
+
+        private readonly List<byte> _byteList = new List<byte>();
```

Comment:
Could you use an `ImmutableList`?

Reply:
Addressed in https://github.com/messerli-informatik-ag/pgp-word-list/pull/1/commits/182ab8e29afaf17e028f101f49b1143780742f7b, https://github.com/messerli-informatik-ag/pgp-word-list/pull/1/commits/84783867218fd7fc5345719729655caeeba65565